src/Main/resources/homework4_part2.sql

```diff
@@ -0,0 +1,77 @@
+select title from cinema.movies;
+
+select director from cinema.movies;
+
+select title, director from cinema.movies;
+
+select title, year from cinema.movies;
+
+select * from cinema.movies;
+
+select title from cinema.movies
+where year between 2000 and 2010;
+
+select title from cinema.movies
+where year not between 2000 and 2010;
+
+select title, year from cinema.movies
+where year in (2000, 2010, 2020);
+
+select title from cinema.movies
```

Comment:
Find the movies with title starting with either "b", "s", or "p"
you can implement it with 1 query